src/ripple/shamap/SHAMap.h

```diff
@@ -82,7 +82,7 @@ class SHAMap
     Family&                         f_;
     beast::Journal                  journal_;
     std::uint32_t                   seq_;
-    std::uint32_t                   ledgerSeq_; // sequence number of ledger this is part of
+    std::uint32_t mutable           ledgerSeq_; // sequence number of ledger this is part of
```

Comment:
`ledgerSeq_` is bunk. Its a piece of metadata attached to the `SHAMap` that doesn't belong there. Imagine if `std::vector` had a data member `std::string description_` where you could put an application defined value. Same thing.